### DISCUSSION TITLE
Access myTourbook from different PCs

### DISCUSSION THREAD

**Discussion by JoerniCarboni** [mytourbook/mytourbook#1410] [General]
Hi community, 
I'm using tourbook and like a lot. Now I want to use it on different PCs in parallel Is it possible to configure the storage of my tours in a way, so it is stored on my private cloud and I can access it from different PCs? If it is possible, would be great, if someone would to tell me how should I do it ;-)
Best JoerniCarboni

**Comment by wolfgang-ch** [comment]
The data directory could be customized https://mytourbook.sourceforge.io/mytourbook/index.php/download-install/data-directory but only **one** MT app can access these data at the same time, mainly the tour database is managing this lock

**Comment by JoerniCarboni** [comment]
Hi Wolfgang,
thanks a lot for the information.Starting with one question further are
popping up , sorry for that.

When looking into the config file I see the following:

#This configuration file was written by:
org.eclipse.equinox.internal.frameworkadmin.equinox.EquinoxFwConfigFileParser
#Sat Jan 27 09:30:20 CET 2024
eclipse.product=net.tourbook.productWithJRE
osgi.splashPath=platform\:/base/plugins/net.tourbook
org.eclipse.equinox.simpleconfigurator.configUrl=file\:org.eclipse.equinox.simpleconfigurator/
bundles.info
osgi.bundles.defaultStartLevel=4
eclipse.application=net.tourbook.appWithJRE
***@***.***/mytourbook*
***@***.***
\:start
***@***.***/../p2*
eclipse.p2.profile=DefaultProfile
osgi.framework=file\:plugins/org.eclipse.osgi_3.18.400.v20230509-2241.jar
osgi.framework.extensions=reference\:file\:org.eclipse.osgi.compatibility.state_1.2.800.v20221116-1440.jar

My first question is which is the right line to modify? I have marked the
two lines, which I think, in bold. Could you guide me here?

My second question is, what is the specific file name and location, where
all my tours are stored? Is it just one file I have to move to my cloud
space and have then two local installations, where I configure the both
config.ini files identically or are there different files to be moved to
the cloud? My first idea was to install the whole tool in the cloud, but I
was not able to select the right location during the installation process.

Best JoerniCarboni

Am Fr., 11. Okt. 2024 um 07:12 Uhr schrieb Wolfgang Schramm <
***@***.***>:

**Reply by wolfgang-ch** [reply]
I would recommend to use the commandline parameter as you don't have to modify config.ini on each new MT update, otherwise you have to modify the content in the red box

![image](https://github.com/user-attachments/assets/96c341a7-d5e7-4392-84bf-5b1edffee64a)

All files/folders in the data folder are needed for MT

**Comment by JoerniCarboni** [comment]
Hi Wolfgang,

thanks a lot for your patience and explanation. It took a little while, but
now it is working by using the command line parameter :😃.
Nevertheless to mention that it is quite cool tool !

Best JoerniCarboni

Am Fr., 11. Okt. 2024 um 14:33 Uhr schrieb Wolfgang Schramm <
***@***.***>:
